packages/expo-navigation-bar/package.json

```diff
@@ -33,7 +33,6 @@
   "license": "MIT",
   "homepage": "https://docs.expo.dev/versions/latest/sdk/navigation-bar",
   "dependencies": {
-    "@react-native/normalize-colors": "0.74.84",
```

Comment:
I don't really think this module is going to change much in the future. Can't we just use a looser version lock?

Reply:
the problem is that even this module doesn't change, react-native will publish new versions every releases. so people could possibly have multiple @react-native/normalize-colors.

for example, we use @react-native/normalize-colors@0.74.84 in expo-navigation-bar, if people trying newer react-native before we publish newer expo-navigation-bar, then it could pull both @react-native/normalize-colors@0.74.84 and 0.74.85.

normalize-colors could be less risky but assets-registry has more impact so i would like to remove all these react-native sub-packages from our dependencies.